src/styles/_body.scss

```diff
@@ -51,3 +51,9 @@ body.dark {
 body.lt-sm.dark {
   background-color: $card-bg-dark;
 }
+
```

Comment:
Even though these _*.scss files are all included, the correct file to add notification-related styles is `_dialogs.scss`. Please, move this code to that file.